product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: "Postgres Enterprise Manager 9.6.0 release notes"
+navTitle: Version 9.6.0
+---
+
+Released: 15 May 2024
+
+New features, enhancements, bug fixes, and other changes in PEM 9.6.0 include:
+
+| Type         | Description                                                                                                                                                                                                                                         |
+|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enhancement  | Added an option to enable the probes using the pemworker utility.                                                                                                                                                                                   |
+| Enhancement  | Fixed an issue to include the error message in the email of the PEM agent job output. The default character limit for error output is 1000, which can be configured now using `job_notification_error_limit` parameter in the server configuration. |
+| Enhancement  | Added two alert templates &mdash; `Inactive replication slots` and `Conflicting replication slots` and added two probes &mdash; `Replication slots` and `Multixact ID Exhaustion (Wraparound)`.                                                                                                |
+| Bug&nbsp;fix | Fixed an issue where audit manager was setting `edb_audit_statement` parameter to '' instead on 'none' when log statement parameter was left empty in the GUI resulting in failure of server restart.                                               |
+| Bug&nbsp;fix | Updated the default parameter for configuring PEM to use the `Fully Qualified Domain Name (FQDN)` hostname while agent registration.                                                                                                                |
+| Bug&nbsp;fix | Fixed an issue with error `NoneType` has no `len()` when alerts were listed on the alert dashboard.                                                                                                                                                 |
+| Bug&nbsp;fix | Fixed an issue for team role feature for the PEM agents.                                                                                                                                                                                            |
```

Comment:
This is very vague. It's PEM-5109. Can we get a better release note from the eng team?